core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitor.scala

```diff
@@ -378,9 +379,9 @@ class AvroUpsertMonitorActor(
         case Success(entityList) => entityList.size
       }.sum
 
-      val failureReports:List[RawlsErrorReport] = upsertResults collect {
-        case Failure(regrets:RawlsExceptionWithErrorReport) => regrets.errorReport.causes
-      } flatten
+      val failureReports: List[RawlsErrorReport] = upsertResults collect {
+        case Failure(regrets:RawlsExceptionWithErrorReport) => regrets.errorReport
+      }
```

Comment:
This change is because sometimes there are no causes in the error report, which leads to the error report not being added at all. This does mean that we're not reporting back every cause of every error, but instead reporting the top level error message. 

Another solution might be to do both - add both the top level message and the underlying causes.